docs/src/design/decoder/constraints.md

```diff
@@ -18,15 +18,16 @@ AIR constraints for the decoder involve operations listed in the table below. Fo
 | `REPEAT`  | $f_{repeat}$  | 4      | Top stack element is dropped.                                                                    |
 | `SPAN`    | $f_{span}$    | 5      | Stack remains unchanged.                                                                         |
 | `RESPAN`  | $f_{respan}$  | 4      | Stack remains unchanged.                                                                         |
+| `DYN`     | $f_{dyn}$     | 5      | Stack remains unchanged.                                                                         |
 | `CALL`    | $f_{call}$    | 4      | Top stack element is dropped.                                                                    |
 | `SYSCALL` | $f_{syscall}$ | 4      | Top stack element is dropped.                                                                    |
```

Comment:
Not related to this PR, but I think there might be an error here for `CALL` and `SYSCALL` as I don't think we drop the top stack element for this operations. Could you check this?

Reply:
Yeah, they seem to leave the stack unchanged. I'll update it in a separate PR